server/config/production.json

```diff
@@ -5,5 +5,8 @@
   "csp": {
     "enabled": true,
     "reportOnly": true
+  },
+  "babel": {
```

Comment:
for example leave the code the same and the middleware will not do babel stuff if user agent is Firefox?